Sources/Datadog/Core/DatadogFeatureRegistry.swift

```diff
@@ -0,0 +1,62 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+
+/// Defines a shared feature registry for global usage.
+internal enum DatadogRegistry {
+    /// The default feature registry for global usage.
+    ///
+    /// Instances complying to `DatadogFeatureRegistry` can be used
+    /// as a global feature registry. By default, `.default` returns a no-op
+    /// registery.
+    internal static var `default`: DatadogFeatureRegistry = NOOPDatadogRegistry()
+}
```

Comment:
This could also be
```swift
internal var DatadogDefaultRegistry: DatadogFeatureRegistry = NOOPDatadogRegistry()
```
`enum DatadogRegistry` is only a namespace here.